File: ComplementApp.API/Dtos/RadicadoDto.cs
using System;

namespace ComplementApp.API.Dtos
{
    public class RadicadoDto
    {
        public int Consecutivo { get; set; }
        public int PlanPagoId { get; set; }
        public DateTime FechaRadicadoSupervisor { get; set; }
        public string FechaRadicadoSupervisorDescripcion { get; set; }
        public string Estado { get; set; }
        public string Crp { get; set; }
        public string NIT { get; set; }
        public string NombreTercero { get; set; }
        public string NumeroRadicadoProveedor { get; set; }
        public string NumeroRadicadoSupervisor { get; set; }
        public decimal ValorAPagar { get; set; }
        public string Obligacion { get; set; }
        public string OrdenPago { get; set; }
        public DateTime? FechaOrdenPago { get; set; }
        public string FechaOrdenPagoDescripcion { get; set; }
        public string TextoComprobanteContable { get; set; }

    }
}

File: ComplementApp-SPA/src/app/_dto/radicadoDto.ts
export interface RadicadoDto {
  consecutivo: number;
  planPagoId: number;
  fechaRadicadoSupervisor: Date;
  fechaRadicadoSupervisorDescripcion: string;
  estado: string;
  crp: string;
  nit: string;
  nombreTercero: string;
  numeroRadicadoProveedor: string;
  numeroRadicadoSupervisor: string;
  valorAPagar: number;
  obligacion: string;
  ordenPago: string;
  fechaOrdenPago?: Date;
  fechaOrdenPagoDescripcion: string;
  TextoComprobanteContable: string;
}
